src/wizard/Step1/components/PermissionsModal.vue

```diff
@@ -1,11 +1,209 @@
 <template>
-  <div>Permissions Modal</div>
+  <div class="portfolio-managers-modal">
+    <v-dialog
+      v-model="isDialogOpen"
+      max-width="800px"
+      persistent
+      content-class="v-dialog v-dialog--active v-dialog--persistent portfolio-managers-modal-dialog"
+      class="portfolio-managers-modal-dialog"
+    >
+      <v-form ref="formAddManagers">
+        <v-card>
+          <v-card-title>
+            <h2 class="h2">Invite Portfolio Manager</h2>
+            <p>
+              Portfolio Managers can have different levels of access based on
+              their roles within your organization. Invite multiple people with
+              the same permissions at once.
+            </p>
+            <br />
+          </v-card-title>
+          <v-card-text>
+            <label class="form-field-label"> Email address </label>
+            <div class="form-field-group">
+              <div
+                class="form-field-item"
+                v-for="(manager, counter) in listManagers"
+                v-bind:key="counter"
+              >
+                <v-text-field
+                  outlined
+                  dense
+                  height="42"
+                  type="email"
+                  :rules="validEmailRules"
+                  :messages="['Messages']"
+                  v-model.lazy="listManagers[counter]"
+                  placeholder="add e-mail"
+                  hide-details="true"
+                  class="invite-portfolio-manager-email"
+                >
+                </v-text-field>
```

Comment:
the email field should look like this...
![image](https://user-images.githubusercontent.com/84856468/128226338-b8704d67-edde-41f6-8bc1-fc5c109bbe2c.png)

remove trashcan and add 'x' on the outside of it.